docs/restructure-phases/phase-c-report.md

```
@@ -0,0 +1,497 @@
+// ReactionHandlerDeps — injected at runtime:
+interface ReactionHandlerDeps {
+  sessionManager: SessionManager;
+  config: OrchestratorConfig;
+  registry: PluginRegistry;
+  notifyHuman: (event: OrchestratorEvent, priority: EventPriority) => Promise<void>;
+  createEvent: (type: EventType, opts: EventOpts) => OrchestratorEvent;
+}
+```
+
+**Risk Assessment:**
+| Risk | Severity | Mitigation |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)